src/parsers/manifest/dash/node_parsers/MPD.ts

```diff
@@ -25,13 +28,18 @@ import {
   parseScheme,
 } from "./utils";
 
+interface IBaseURL {
```

Comment:
Instead of replicating this everywhere, we could do the same thing that `IScheme` and `parseScheme` does